theia-rust-docker/next.package.json

```diff
@@ -0,0 +1,51 @@
+{
+    "private": true,
```

Comment:
Consider adding a name for this Theia application. See e.g.: [here](https://github.com/theia-ide/theia-apps/pull/204/files#diff-5dc95d46717c4be6967fb46980c2f064R3). 

I am not sure if the way the user's home directory is mounted in the image if it might make sense to enable the "thrash" can for deleted files? If so you can omit this preference since the default is enabled (true).

Reply:
Added